fiftyone/utils/depth_anything.py

```
@@ -0,0 +1,314 @@
+        )
+
+
+da3_api = fou.lazy_import(
+    "depth_anything_3.api",
+    callback=_ensure_depth_anything_3,
+)
+
+DEFAULT_DA3_MODEL = "depth-anything/da3-base"
+
+
+def _normalize_float_to_uint8(img_array):
```

Ruff (0.14.10)

Missing return type annotation for private function `_normalize_float_to_uint8`

(ANN202)